.github/workflows/hqs-build-deploy-rust-pyo3.yml

```diff
@@ -0,0 +1,58 @@
+name: HQS build and deploy for rust pyo3 repos
+
+on:
+  workflow_dispatch:
+  release:
+    types: [created]
+
+jobs:
+
+  deploy_cratesio:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: Swatinem/rust-cache@v2
+    - uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        default: true
+    - name: setup cargo
+      run: |
+        cargo login ${{ secrets.CRATES_IO_API_TOKEN }}
+        cargo publish --manifest-path=struqture/Cargo.toml
+        sleep 45
+        cargo publish --manifest-path=struqture-py/Cargo.toml
+        sleep 45
+        cargo publish --manifest-path=struqture-py-macros/Cargo.toml
+# Build linux wheels and upload them to PyPi
+  build_maturin_builds_linux:
+    name: maturin_build-linux
+    needs: deploy_cratesio
+    uses: HQSquantumsimulations/reusable_workflows/.github/workflows/reusable_build_maturin_linux.yml@main
+    with: 
+      py_interface_folder: "struqture-py"
+      deploy: true
+      # Whether to build for aarch64. Can fail if there are C/C++ dependencies
+      build_for_arm: true
```

Comment:
The default option on the original workflow is false, is it ok to set it to true?

Reply:
There aren't any C++ dependencies, I think it should be fine